notebooks/official/prediction/get_started_with_raw_predict.ipynb

```diff
@@ -127,186 +130,122 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "install_aip"
+        "id": "3b1ffd5ab768"
       },
       "source": [
-        "## Installation\n",
-        "\n",
-        "Install the following packages to execute this notebook."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": null,
-      "metadata": {
-        "id": "install_aip"
-      },
-      "outputs": [],
-      "source": [
-        "! pip3 install --upgrade --quiet google-cloud-aiplatform \n",
-        "! pip3 install tensorflow-hub"
+        "## Getting Started"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "58707a750154"
+        "id": "aae9ca040eab"
       },
       "source": [
-        "### Colab only: Uncomment the following cell to restart the kernel."
+        "### Install Vertex AI SDK and other required packages\n"
```

Comment:
add "for Python":

- Vertex AI SDK for Python

Reply:
Fixed the issue